<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="action_disp_seconds"/>
    <addaction name="action_cloc_me_menu_settings_regional"/>
    <addaction name="action_dati_ia_adjust_date_and_time"/>
    <addaction name="action_cloc_alarm_settings_title"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="action_sfil_va_number_of_objects_images">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>imageSettings</string>
   </property>
  </action>
  <action name="action_disp_seconds">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>secondsSettings</string>
   </property>
  </action>
  <action name="action_cloc_me_menu_settings_regional">
   <property name="text">
    <string>regSettings</string>
   </property>
  </action>
  <action name="action_dati_ia_adjust_date_and_time">
   <property name="text">
    <string>dateSettings</string>
   </property>
  </action>
  <action name="action_cloc_alarm_settings_title">
   <property name="text">
    <string>alSettings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
